packages/did/src/DidResolver/DidResolver.spec.ts

```diff
@@ -49,39 +45,47 @@ const didWithServiceEndpoints: DidUri = `did:kilt:${addressWithServiceEndpoints}
 const deletedAddress = '4rrVTLAXgeoE8jo8si571HnqHtd5WmvLuzfH6e1xBsVXsRo7'
 const deletedDid: DidUri = `did:kilt:${deletedAddress}`
 
-const didNotFound = ApiMocks.mockChainQueryReturn('did', 'did')
-const encodedDidWithAuthenticationKey = ApiMocks.mockChainQueryReturn(
-  'did',
-  'did',
-  [
-    '01234567890123456789012345678901',
-    [],
-    undefined,
-    undefined,
-    [],
-    '123',
-    [addressWithAuthenticationKey, '0'],
-  ]
-)
 const didIsBlacklisted = ApiMocks.mockChainQueryReturn(
   'did',
   'didBlacklist',
   'true'
 )
 
+const augmentedApi = ApiMocks.createAugmentedApi()
+
 let mockedApi: any
 beforeAll(() => {
   mockedApi = ApiMocks.getMockedApi()
   ConfigService.set({ api: mockedApi })
 
-  mockedApi.query.did.did.mockReturnValue(encodedDidWithAuthenticationKey)
-  mockedApi.query.did.serviceEndpoints.mockReturnValue(
-    ApiMocks.mockChainQueryReturn('did', 'serviceEndpoints', [
-      'foo',
-      ['type-service-1'],
-      ['x:url-service-1'],
-    ])
-  )
+  // Mock `api.rpc.did.query(didUri)`
+  // By default it returns a simple LinkedDidInfo with no web3name and no accounts linked.
```

Comment:
I found that I wished there were some comments about what was mocked before. That is why I thought about adding few. Maybe it does not give much value to you since you worked on this, but for me and I guess for others it might be helpful.

Reply:
That’s fair. I mostly was surprised by repeated "returns something" below.